app/src/main/java/com/codingblocks/cbonlineapp/mycourse/content/PdfActivity.kt

```diff
@@ -86,6 +89,18 @@ class PdfActivity : BaseCBActivity(), AnkoLogger {
         this@PdfActivity.registerReceiver(receiver, intentFilter)
     }
 
+    override fun onSaveInstanceState(outState: Bundle, outPersistentState: PersistableBundle) {
+        super.onSaveInstanceState(outState, outPersistentState)
+        outState.putString(CONTENT_ID, intent.getStringExtra(CONTENT_ID))
+        outState.putString(SECTION_ID, intent.getStringExtra(SECTION_ID))
+    }
+
+    override fun onRestoreInstanceState(savedInstanceState: Bundle) {
+        super.onRestoreInstanceState(savedInstanceState)
+        contentID = savedInstanceState.getString(CONTENT_ID).toString()
```

Comment:
What's the purpose of saving them if are not being used or will be overwritten by null values

Reply:
@aggarwalpulkit596 Bhaiya are you talking about sectionId? Because that is the only thing not being used in PDFActivity.

Reply:
I'm talking about both of the values because these values will be overriden in onCreate no matter what is the value so i don't see any point why they should be saved in savedInstanceState